docs/How-to-release.md

```diff
@@ -119,99 +130,99 @@ Guide to build the release from source :
-
-```shell
-export SVN_EDITOR=vim
-svn mv https://dist.apache.org/repos/dist/dev/skywalking/cli/$VERSION https://dist.apache.org/repos/dist/release/skywalking/cli
-# ....
-# enter your apache password
-# ....
-
-```
+1. **Send the closing vote mail to announce the result**.  When count the binding and no binding votes, please list the names of voters. An example like this:
 
-1. Public download links of source and distribution tar/zip locate in `http://www.apache.org/dyn/closer.cgi/skywalking/cli/$VERSION/`.
-We only publish Apache mirror path as release info.
+   ```
+   [RESULT][VOTE] Release Apache SkyWalking CLI version $VERSION
+   
+   3 days passed, we’ve got ($NUMBER) +1 bindings:
+   xxx
+   xxx
+   xxx
+   ...
+   (list names)
+    
+   I’ll continue the release process.
+   ```
 
-1. Public asc and sha512 locate in `https://www.apache.org/dist/skywalking/cli/$VERSION/xxx`
 
-1. Public KEYS pointing to  `https://www.apache.org/dist/skywalking/KEYS`
+## Publish release
 
-1. Send ANNOUNCEMENT email to `dev@skywalking.apache.org`.
+1. Move source codes tar balls and distributions to `https://dist.apache.org/repos/dist/release/skywalking/`, **you can do this only if you are a PMC member**.
 
-1. Update links on the website download page. http://skywalking.apache.org/downloads/ . Include the new source codes, distribution packages, corresponding sha512, asc and document
-links. Links could be found by following rules(3)-(6).
+    ```shell
+    export SVN_EDITOR=vim
+    svn mv https://dist.apache.org/repos/dist/dev/skywalking/cli/$VERSION https://dist.apache.org/repos/dist/release/skywalking/cli
+    # ....
+    # enter your apache password
+    # ....
+```
+    
+1. Refer to the previous [PR](https://github.com/apache/skywalking-website/pull/118), update news and links on the website. There are seven files need to modify.
 
-1. Add a release event on website homepage and event page. Announce the public release with changelog or key features.
+1. Update [Github release page](https://github.com/apache/skywalking-cli/releases), follow the previous convention.
 
-1. Send ANNOUNCE email to `dev@skywalking.apache.org` and `announce@apache.org`, the sender should use his/her Apache email account.
+1. Send ANNOUNCE email to `dev@skywalking.apache.org` and `announce@apache.org`, the sender should use his/her Apache email account. You can get the permlink of vote thread at [here](https://lists.apache.org/list.html?dev@skywalking.apache.org).
 
 ```text
 Subject: [ANNOUNCEMENT] Apache SkyWalking CLI $VERSION Released
 
 Content:
 
-Hi all,
+Hi the SkyWalking Community
 
-The Apache SkyWalking Team is glad to announce the release of Apache SkyWalking CLI $VERSION.
+On behalf of the SkyWalking CLI Team, I’m glad to announce that SkyWalking CLI $VERSION is now released.
 
-SkyWalking CLI: A Command Line Interface for Apache SkyWalking https://skywalking.apache.org/
+SkyWalking CLI: A command line interface for SkyWalking.
```

Comment:
You should not remove `Apache` before `SkyWalking`. It is not recommended.

Reply:
reverted the change